application/api/api_test.go

```diff
@@ -292,3 +297,103 @@ func sendJSONRPCRequest(rpcAddress string, jsonReq string) (string, error) {
 
 	return string(body), nil
 }
+
+func TestDefaultRPC_Integration_GetAppBlock(t *testing.T) {
```

Comment:
@b00ris  you mentioned, integration test is enough for testing RPC method, isn't it?